### PR TITLE
Specific error message when specifying path as a volume source

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@docker.com>

**What I did**
add specific error message when user tries to specify a host path in compose file

**Related issue**
https://github.com/docker/compose-cli/issues/865

/test-aci

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

File: aci/convert/volume.go [hunk 1]
```diff
@@ -82,8 +82,11 @@ func (p projectAciHelper) getAciFileVolumes(ctx context.Context, helper login.St
 func (s serviceConfigAciHelper) getAciFileVolumeMounts(volumesCache map[string]bool) ([]containerinstance.VolumeMount, error) {
 	var aciServiceVolumes []containerinstance.VolumeMount
 	for _, sv := range s.Volumes {
+		if strings.Contains(sv.Source, "/") {
```
Comment (ndeloof) [inline aci/convert/volume.go:85]: better rely on sv.Type as https://github.com/compose-spec/compose-go/blob/5afecaa4cb35039bebfc86ae0c3742dab7cae0f3/loader/volume.go#L121 do already handle filepath detection for source.

File: aci/convert/volume.go [hunk 2]
```diff
@@ -82,8 +82,11 @@ func (p projectAciHelper) getAciFileVolumes(ctx context.Context, helper login.St
...
+			return []containerinstance.VolumeMount{}, fmt.Errorf("host path (%q) not allowed as volume source, you need to reference an Azure File Share defined in the 'volumes' section", sv.Source)
+		}
 		if !volumesCache[sv.Source] {
-			return []containerinstance.VolumeMount{}, fmt.Errorf("could not find volume source %q", sv.Source)
+			return []containerinstance.VolumeMount{}, fmt.Errorf("could not find volume source %q, service volumes must reference volumes defined in the 'volumes' section", sv.Source)
```
Comment (ndeloof) [inline aci/convert/volume.go:89]: already checked by https://github.com/compose-spec/compose-go/blob/5afecaa4cb/loader/validate.go#L38

**Review by ndeloof — CHANGES_REQUESTED** [review]
could better leverage compose-go to parse and validate volume specs

File: aci/convert/volume.go (hunk 1 above)
Comment (gtardif) [reply aci/convert/volume.go]: Thx! updated

File: aci/convert/volume.go (hunk 2 above)
Comment (gtardif) [reply aci/convert/volume.go]: fixed

**Review by ndeloof — APPROVED** [review]

**Pull request merged**